utils/src/deployer.rs

```diff
@@ -307,6 +307,143 @@ pub async fn deploy_mock_light_client_contract<M: Middleware + 'static>(
     Ok(contract.address())
 }
 
+/// Deployment `LightClientMock.sol` as proxy for testing
+pub async fn deploy_light_client_contract_as_proxy_for_test<M: Middleware + 'static>(
```

Comment:
It looks like this should be feature gated with `#[cfg(any(test, feature = "testing"))]`

Reply:
good call! Thank you!

Reply:
that function appeared to be duplicated as it was already in a test module . Removed duplicates here: https://github.com/EspressoSystems/espresso-sequencer/pull/2081/commits/58a989045daef84ca8a7c1e34b7209ae557a1bba